src/components/Items/Question/Body.vue

```diff
@@ -76,8 +73,36 @@ export default {
       default: false,
       type: Boolean,
     },
+    previewMode: {
```

Comment:
I think preview is how we are using it and should not be tied to the component - can be `manualDisable` or something else - not tied to how we are using it.